hack/e2e.sh

```diff
@@ -89,17 +99,28 @@ PE_URL=$(oc get route cincinnati-policy-engine -o jsonpath='{.spec.host}')
 export GRAPH_URL="http://${PE_URL}/api/upgrades_info/v1/graph"
 
 # Wait for route to become available
-ATTEMPTS=10
 DELAY=10
-
-while [ $ATTEMPTS -ge 0 ]; do
+for i in $(seq 10); do
   CODE=$(curl -s -o /dev/null -w "%{http_code}" --header 'Accept:application/json' "${GRAPH_URL}?channel=a")
   if [ "${CODE}" == "200" ]; then
     break
-  else
-    sleep ${DELAY}
-    ATTEMPTS=$((ATTEMPTS-1))
-  fi
```

Comment:
[e2e][1]:

```
hack/e2e.sh: line 109: syntax error near unexpected token `done'
```

You need to keep this closing `fi`.

[1]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_cincinnati/248/pull-ci-openshift-cincinnati-master-e2e/363/build-log.txt